### ISSUE TITLE
eHost to brat format

### ISSUE THREAD

**Issue by fredymad** [nlplab/brat#1389]
Hi,

Is there any available converter to change txt.knowtator.xml from eHost to .ann?

**Comment by ghost** [comment]
No, not that we are aware of. It may help to link the format, although I doubt any of the “core” devs have cycles to commit.
